lib/manageiq/rpm_build/rpm_repo.rb

```diff
@@ -70,6 +71,15 @@ def update
               client.delete_object(:bucket => OPTIONS.rpm_repository.s3_api.bucket, :key => object.key)
             end
           end
+
+          # Bust the cache for updated files
+          if OPTIONS.rpm_repository.digitalocean_access_token
+            puts "Purging the cache for files that were uploaded"
+            require 'droplet_kit'
```

Comment:
@bdunne I just realized this isn't in the Gemfile...is that intentional?

Reply:
No, apparently I forgot to save the Gemfile change... https://github.com/ManageIQ/manageiq-rpm_build/pull/72